osfoffline/application/main.py

```diff
@@ -163,6 +165,14 @@ def quit(self):
             logger.info('Quitting application')
             QApplication.instance().quit()
 
+    def sync_now(self):
+        if not self.background_worker:
+            self.start()
+        if self.background_worker.is_alive():
```

Comment:
As discussed on another PR, `self.background_worker.is_alive` already seems to be called in `self.pause`.

Also, it seems there might be a race condition if the background worker stops in between the (first/second) check and the stop action. (EAFP might be a better way to go)